test/unit/processing/confirm-due-dates.test.js

```diff
@@ -0,0 +1,588 @@
+const { AP, AR } = require('../../../app/ledgers')
+const confirmDueDates = require('../../../app/processing/confirm-due-dates')
+const moment = require('moment')
+
+describe('confirm due dates', () => {
+  test('should return no change if no previous requests', () => {
+    const paymentRequests = [{
+      ledger: AP,
+      dueDate: '09/11/2022',
+      schedule: 'Q4',
+      value: -100
+    }]
+    const confirmedPaymentRequests = confirmDueDates(paymentRequests)
+    expect(confirmedPaymentRequests[0].dueDate).toBe('09/11/2022')
+    expect(confirmedPaymentRequests[0].schedule).toBe('Q4')
+  })
+
+  test('should return no change if no previous requests as empty array', () => {
+    const paymentRequests = [{
+      dueDate: '09/11/2022',
```

Comment:
This should be based off `moment(new Date()).add(... something waaayyy in the future)` otherwise this will break in Nov and we dnt want to scare those maintaining it then

Reply:
No it won't, ignore me